test cases/unit/130 wrap file sftp/Makefile

```diff
@@ -0,0 +1,2 @@
+foo.tar.gz: foo/foo.c foo/foo.h foo/meson.build
+	tar -c -z -f $@ $^
```

Comment:
Tests are not allowed to use _any_ Unix tools. This functionality needs to be implemented with a Python script instead.

Reply:
The test doesn't use this Makefile as far as I can tell. It's just there to serve as executable documentation on how the submitter produced the tarball that is checked into git.

Reply:
The Makefile is only there to make it reasonable to modify this in the future, for whatever reason. The test also calculates the hash for the wrap file, so that one doesn't need to modify the test code if the archive is updated.

Reply:
Also I'm pretty sure relying on OpenSSH is a bigger hindrance to Windows users lacking posix tools.

Tar actually does ship natively on sufficiently new windows, though. Microsoft started shipping both tar and curl these days.